src/stickers.yml

```diff
@@ -23,6 +23,287 @@
+  tags:
+  - telegram
+  - animals
+  - sabretooth
+  - for children
+
+3461f4aed915ebbdca5e6ffde36568c6:
+  animated: true
+  key: 809d3e023f438d046e01bb9b543c3205e2284d19ddf2f2c355a72181a3617721
+  source: https://t.me/addstickers/BananaAnimated
+  tags:
+  - telegram
+  - banana
+  - fruit
+  - rainbow
+
+34b0a0615402dc43362623052eef9e23:
+  animated: true
+  key: 68545ef2d25cc0e90ce5e7dc1d0ac874e2036c6bb4681965dff69e9ee6ed6df0
+  source: https://t.me/addstickers/cupiman
+  tags:
+  - telegram
+  - cupid
+  - valentine's day
+
+4206c9a65ee27228745a9b2673c553ba:
+  animated: true
+  key: e5c2ac0358eed6baff1fe0e161bdacb0ecbec855107df7f56cdd0a9e6cfde2dd
+  source: https://t.me/addstickers/GlaxAnimated
+  tags:
+  - telegram
+  - furry
+  - dragon
+  - glax
+  - mattbass
+  - meme
+
+4be2c93cda1e4b13c44fa16ff6be155a:
+  animated: true
+  key: 6350cfc5a3502b9f5ff2460e40aa0a3d8df5a4df91ed664972503a3e33d24c0d
+  source: https://t.me/addstickers/ToiletPaperBoy
+  tags:
+  - telegram
+  - toilet paper
+
+4d049350cb3adf152c03670d559ad2e6:
+  animated: true
+  key: c6fbb740042c6111696b67fedb367a12b77aecae5932c3161b5568825945f511
+  source: https://t.me/addstickers/PokeAnimated
+  tags:
+  - telegram
+  - pokemon
+  - for children
+
+585caf6cbbfd854f49bd734fb7f50441:
+  animated: true
+  key: 45bc33e475d881a55837e64f4ed65638f5b5fcf61e870c5cb4d7ac2888bcb121
+  source: https://t.me/addstickers/KangarooFighter
+  tags:
+  - telegram
+  - animals
+  - for children
+  - kangaroo
+
+67d3dfefcd64c994bd8f47c4ceb80548:
+  animated: true
+  key: ea874154ff436596e7ee6b6a30a047d52c897f70ebee129030e9cfc9aaac7915
+  source: https://t.me/addstickers/MrPugDog
+  tags:
+  - telegram
+  - animals
+  - pug
+  - dog
+  - for children
+
+71ae2a3f7c772f0948f4b9ee8d1d17ea:
+  animated: true
+  key: 544b290c0f949454ce5d1180ae54837658cbe9c32af211824de9589b9ee327f2
+  source: https://t.me/addstickers/StarPatrick
+  tags:
+  - telegram
+  - spongebob
+  - patrick
+  - patrick star
+  - for children
+
+7483b3e2302cad1a6a9c1237739427c8:
+  animated: true
+  key: c13027adae00438874043ad988f4a291eb029f8f55e3fbe1016e9741fff3064e
+  source: https://t.me/addstickers/MrSeal
+  tags:
+  - telegram
+  - animals
+  - seal
+  - for children
+
+803753d332b97d06aee34485882a5dc8:
+  animated: true
+  key: 83366449dc3147d1feccf307e1c41542ca9174ed6ced7137cbc11a0155478deb
+  source: https://t.me/addstickers/MiladyNoir
+  tags:
+  - telegram
+  - broken on desktop
+  - noir
+
+811a1fdaecc619a5be5f8423a6ca256f:
+  animated: true
+  key: e7d26e5ef6c7c4d57c2dbc806056f572533818393b5a8590b4466c9759158017
+  source: https://t.me/addstickers/Friendly_Panda
+  tags:
+  - telegram
+  - animals
+  - panda
+  - for children
+
+8a5f2c209203858a31c3f838074c726f:
+  animated: true
+  key: ab7e759fc6b45adfd151984e2a9c5b914f624e9684c08eb251930722a41c76d6
+  source: https://t.me/addstickers/RedShibaInu
+  tags:
+  - telegram
+  - doge
+  - meme
+  - dog
+  - animal
+  - shiba inu
+
+8b06294d286957bd35acb8b9da861598:
```

Comment:
We already have a lot of Pusheen stickers, and most of the stickers in this packs are already on the site. Could you remove it?

Reply:
Pusheen removed.